src/Workspaces/Core/Portable/TemporaryStorage/TrivialTemporaryStorageService.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis;
+
+internal sealed class TrivialTemporaryStorageService : ITemporaryStorageServiceInternal
```

Comment:
adapted from the old version
https://github.com/dotnet/roslyn/blob/ea35f7ce524e9424adb16fd6b8a2308e55e6de0e/src/Workspaces/Core/Portable/Workspace/Host/TemporaryStorage/TrivialTemporaryStorageService.cs